### PR TITLE
feat: add provider fields

### PR DESCRIPTION
**Pull request by peterdeme**
Add new fields

### CODE REVIEW

File: src/main/java/io/getstream/chat/java/models/App.java [hunk 1]
```diff
@@ -763,6 +763,14 @@ public AppGetRateLimitsRequest endpoint(@NotNull String endpoint) {
     @JsonProperty("skip_devices")
     private Boolean skipDevices;
 
+    @Nullable
+    @JsonProperty("push_provider_name")
+    private String pushProviderName;
+
+    @Nullable
+    @JsonProperty("push_provider_type")
+    private String pushProviderType;
```
Comment (driver-devel) [inline src/main/java/io/getstream/chat/java/models/App.java:772]: Should be PushProviderType here once previous PR is merged

Comment (peterdeme) [reply src/main/java/io/getstream/chat/java/models/App.java]: ✅

**Review by ferhatelmas — CHANGES_REQUESTED** [review]
needs provider name in `DeviceError`

**Review by driver-devel — DISMISSED** [review]
LGTM

File: src/main/java/io/getstream/chat/java/models/App.java [hunk 2]
```diff
@@ -794,6 +798,14 @@ public AppGetRateLimitsRequest endpoint(@NotNull String endpoint) {
     @JsonProperty("skip_devices")
     private Boolean skipDevices;
 
+    @Nullable
+    @JsonProperty("push_provider_name")
+    private String pushProviderName;
+
+    @Nullable
+    @JsonProperty("push_provider_type")
+    private PushProvider.PushProviderType pushProviderType;
```
Comment (ferhatelmas) [inline src/main/java/io/getstream/chat/java/models/App.java:807]: seems we're duplicating this between device and app where device one is missing huawei and xiaomi

**Review by ferhatelmas — CHANGES_REQUESTED** [review]
fix provider type under device

File: src/main/java/io/getstream/chat/java/models/App.java (hunk 2 above)
Comment (peterdeme) [reply src/main/java/io/getstream/chat/java/models/App.java]: unified ✅

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Pull request merged**